packages/block-library/src/navigation/edit/use-go-to-overlay-editor.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * WordPress dependencies
+ */
+import { privateApis as routerPrivateApis } from '@wordpress/router';
+
+/**
+ * Internal dependencies
+ */
+import { unlock } from '../../lock-unlock';
+
+const { useHistory } = unlock( routerPrivateApis );
+
+export default function useGoToOverlayEditor() {
+	const history = useHistory();
+
+	function goToOverlayEditor( overlayId, navRef ) {
+		history.push( {
```

Comment:
This API has be removed it seems. I cannot seem to get `history.navigate` to work 😕